it/v2/endpoints/ListPaymentsAndAllocationDetailsControllerISpec.scala

```diff
@@ -163,9 +165,10 @@ class ListPaymentsAndAllocationDetailsControllerISpec extends IntegrationBaseSpe
         (BAD_REQUEST, "INVALID_DATE_FROM", BAD_REQUEST, FromDateFormatError),
         (BAD_REQUEST, "INVALID_DATE_TO", BAD_REQUEST, ToDateFormatError),
         (BAD_REQUEST, "INVALID_DATE_RANGE", BAD_REQUEST, RuleInvalidDateRangeError),
-        (FORBIDDEN, "REQUEST_NOT_PROCESSED", INTERNAL_SERVER_ERROR, InternalError),
+        (BAD_REQUEST, "INVALID_REQUEST", BAD_REQUEST, RuleInconsistentQueryParamsErrorListSA),
+        (BAD_REQUEST, "REQUEST_NOT_PROCESSED", BAD_REQUEST, BadRequestError),
         (NOT_FOUND, "NO_DATA_FOUND", NOT_FOUND, NotFoundError),
-        (UNPROCESSABLE_ENTITY, "PARTIALLY_MIGRATED", INTERNAL_SERVER_ERROR, InternalError),
+        (BAD_REQUEST, "PARTIALLY_MIGRATED", BAD_REQUEST, BadRequestError),
```

Comment:
I think this should be UNPROCESSABLE_ENTITY instead of BAD_REQUEST ?

Reply:
4XX should be mapped to 4XX, so in this case 400 is correct